conda_concourse_ci/compute_build_graph.py

```diff
@@ -203,9 +203,13 @@ def add_recipe_to_graph(recipe_dir, graph, run, worker, conda_resolve,
             continue
 
         name = package_key(metadata, worker['label'], run)
+        noarch_python = metadata.noarch == 'python'
```

Comment:
what about noarch: generic?  We should be able to collapse it also, right?

Reply:
I think so but I could find any examples where noarch: generic is used outside of CDT packages.

Reply:
I believe the plan is to employ it somehow for R stuff.  It also gets used once in a while for data-only packages, like nltk-data

Reply:
Added support for `noarch: generic` and `noarch: True`